openmdao/core/driver.py

```diff
@@ -79,20 +113,42 @@ def _setup_driver(self, problem):
         self._objs = model.get_objectives(recurse=True)
         self._cons = model.get_constraints(recurse=True)
 
-    def get_design_var_values(self):
+        self._rec_mgr.startup(self)
+        if (self._rec_mgr._recorders):
+            self._model_viewer_data = _get_viewer_data(problem)
+        self._rec_mgr.record_metadata(self)
+
+    def get_design_var_values(self, filter=None):
         """
         Return the design variable values.
 
         This is called to gather the initial design variable state.
 
+        Parameters
+        ----------
+        filter : list
+            List of desvar names used by recorders to
+            filter by includes/excludes.
```

Comment:
Yeah, I guess that's what I was getting at.  My comment could have been clearer.  I was basically just suggesting to get rid of the '/excludes'.